content/en/internal/configuration-database/_index.md

```diff
@@ -1,44 +1,27 @@
 ---
 title: "Configuration Database"
-description: "Configuration Database (GOCDB) documentation"
-weight: 30
+description: "Service configuration database"
```

Comment:
I don't feel like Service configuration database is a good description for this service.

Reply:
Then please suggest a better description (page subtitle)

Reply:
Maybe something around Information about the topology of the infrastructure?